crates/rust-analyzer/Cargo.toml

```diff
@@ -45,6 +45,10 @@ tracing-log = "0.1.3"
 tracing-tree = "0.2.1"
 always-assert = "0.1.2"
 
+# scip bindings
+scip = { git = "https://github.com/sourcegraph/scip", rev = "552e26c9" }
+protobuf = "3"
```

Comment:
yes, it's just so that we can serialize the output. If you're very opposed to having the protobuf as a dependency, we could potentially serialize it as JSON (and then process that outside of the tool). I think that works (although I'm not 100% sure tbh)

Reply:
Hm, if `scip` can be emitted as JSON as well, that would be helpful, yeah! 

Long-term, I can totally see us emitting protobuf directly, as that's more efficient, but, if we can minimize dependencies at the start, that would be very helpful!